### DISCUSSION TITLE
HOW to ajax non-exist resource without printing 404 (NOT FOUND) on console?

### DISCUSSION THREAD

**Discussion by haelue** [jquery/jquery#4920] [Q&A] [answered]
I am using $.ajax() to get a url resource, but I dont't know whether it's existing, I just want to TEST whether it's existing or not.
This exist-testing is in my code-logic, so I don't want to print "404(NOT FOUND)" on my console. How can I do that?
Many thanks!

NOTE: **$.ajax() print "404(NOT FOUND)" before all success/error/complete callback**

**Comment by mgol** [comment, marked as answer]
I think it's not jQuery that prints those warnings but the browser itself. We can't do anything about that.
